invenio_oauthclient/views/client.py

```diff
@@ -271,3 +271,19 @@ def rest_disconnect(remote_app):
         return _disconnect(remote_app)
     except OAuthRemoteNotFound:
         abort(404)
+
+
+@blueprint.route("/logout")
+def logout():
```

Comment:
can we make the implementation of this endpoint configurable, in case someone would like to implement a different behaviour? (e.g. add extra params?)
How can I, for example, add a redirect back to a specific URL (which might not be the homepage)?

Reply:
This endpoint is only used if the variable `SECURITY_POST_LOGOUT_VIEW=/oauth/logout` is set, meaning that if you want to configure to redirect to another endpoint you just need to register the endpoint and override the `SECURITY_POST_LOGOUT_VIEW` with the new value. To add the redirect back to a specific URL I think it's each remote responsability to do it, for instance in [this case (cds-rdm)](https://github.com/CERNDocumentServer/cds-rdm/pull/66/files#diff-35ea7d0c9f0d3771b33ec0ac671404047bdff444801fcf084c39437c21107ebfR213) you pass the `redirect_uri` pointing to the home page